ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/SciMessageClient.java

```diff
@@ -67,6 +66,58 @@ public CompletableFuture<Void> publish(String topic, JsonElement jsonElement, Pu
     return publishJson(topic, GsonFactory.create().toJson(jsonElement), options);
   }
 
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#request_
+   */
+  public <T> ISubscription request(String topic, Class<T> clazz, ISubscriber<TopicMessage<T>> subscriber) {
+    return requestJson(topic, null, null, clazz, subscriber);
+  }
+
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#request_
+   */
+  public <T> ISubscription request(String topic, Object body, Class<T> clazz, ISubscriber<TopicMessage<T>> subscriber) {
+    return requestJson(topic, GsonFactory.create().toJson(body), null, clazz, subscriber);
+  }
+
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#request_
+   */
+  public <T> ISubscription request(String topic, SciMessageClient.RequestOptions options, Class<T> clazz, ISubscriber<TopicMessage<T>> subscriber) {
+    return requestJson(topic, null, options, clazz, subscriber);
+  }
+
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#request_
+   */
+  public <T> ISubscription request(String topic, Object body, SciMessageClient.RequestOptions options, Class<T> clazz, ISubscriber<TopicMessage<T>> subscriber) {
+    return requestJson(topic, GsonFactory.create().toJson(body), options, clazz, subscriber);
+  }
+
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#publish
+   */
+  public <T> ISubscription request(String topic, JsonElement jsonElement, SciMessageClient.RequestOptions options, Class<T> clazz, ISubscriber<TopicMessage<T>> subscriber) {
+    return requestJson(topic, GsonFactory.create().toJson(jsonElement), options, clazz, subscriber);
+  }
+
+  /**
+   * @see https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#observe_

```

Comment:
should point also to https://scion-microfrontend-platform-api.vercel.app/classes/MessageClient.html#request_